### PR TITLE
cmd: use logger to print errors

### LINKED ISSUE #1786: should display clear message when load with exist meta.
**What happened**:

root@bench-01:~# ./juicefs load sqlite3:///tmp/encoding-test3.db backup.dump 
2022/04/14 17:54:18.469497 juicefs[1008732] <INFO>: Meta address: sqlite3:///tmp/encoding-test3.db [interface.go:383]
2022-04-14 17:54:18.470668 I | Database sqlite3 is not empty

**What you expected to happen**:
 Database sqlite3 is not empty  should be something like "load canceled because database sqlite3:///tmp/encodeing-test3.db is not empty, "
and the log level should be Warning instead of Info.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**

**Environment**:
- JuiceFS version (use `juicefs --version`) or Hadoop Java SDK version:
- Cloud provider or hardware configuration running JuiceFS:
- OS (e.g `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Object storage (cloud provider and region, or self maintained):
- Metadata engine info (version, cloud provider managed or self maintained):
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage):
- Others:

### PR DESCRIPTION
**Pull request by SandyXSD**
close #1786

### CODE REVIEW

File: cmd/bench.go [hunk 1]
```diff
@@ -301,7 +301,7 @@ func bench(ctx *cli.Context) error {
 	setup(ctx, 1)
 	/* --- Pre-check --- */
 	if ctx.Uint("block-size") == 0 || ctx.Uint("threads") == 0 {
-		return os.ErrInvalid
```
Comment (davies) [inline cmd/bench.go:304]: This easier way is to change main():
```
func main() {
	err := Main(os.Args)
	if err != nil {
		log.Fatal(err)
	}
}
```

Comment (SandyXSD) [reply cmd/bench.go:304]: Logger prints the line number and is usually helpful to trace the error. Besides, it's good to use only one way (fatal or return) to handle errors in commands.

Comment (davies) [reply cmd/bench.go:304]: Main() is also in cmd/ package, we can use logger there, right?

Comment (davies) [reply cmd/bench.go:304]: I meant changing `log` to `logger` in main should be enough.

Comment (SandyXSD) [reply cmd/bench.go]: OK

**Pull request merged**